examples/Makefile

```diff
@@ -27,7 +27,6 @@ OFFICIAL_PACKAGES = ./gno.land/p
 OFFICIAL_PACKAGES += ./gno.land/r/demo
 OFFICIAL_PACKAGES += ./gno.land/r/gnoland
 OFFICIAL_PACKAGES += ./gno.land/r/sys
-OFFICIAL_PACKAGES += ./gno.land/r/gov
```

Comment:
Oh, this should have been added from #1284.

BTW, it appears the CI is not following the Makefiles, so you identified the issue by running a command manually. Ideally, we should detect this from the CI.

Reply:
added CI check in [3cf8518](https://github.com/gnolang/gno/pull/2034/commits/3cf8518f9a640ba53cb2e782abda021958d5ae9a)